src/ds/d400/d400-color.cpp

```diff
@@ -216,9 +216,16 @@ namespace librealsense
         }
         else
         {
-            // Work-around for discrepancy between the RGB YUYV descriptor and the parser . Use UYUV parser instead
-            color_ep.register_processing_block(processing_block_factory::create_pbf_vector<uyvy_converter>(RS2_FORMAT_UYVY, map_supported_color_formats(RS2_FORMAT_UYVY), RS2_STREAM_COLOR));
```

Comment:
Was this line not used at all?

Reply:
we have no such format for RGB - UyVy is never supported. YuYv is.